lib/swaps/Swaps.ts

```diff
@@ -258,9 +253,14 @@ class Swaps extends EventEmitter {
 
     const deal: SwapDeal = {
       ...swapRequestBody,
+      takerCurrency,
+      makerCurrency,
+      takerAmount,
+      makerAmount,
       peerPubKey: peer.nodePubKey!,
       localId: taker.localId,
       price: maker.price,
+      isBuy: maker.isBuy,
```

Comment:
Do we need this here if we have the orderID?

Reply:
It's not absolutely necessary but it follows the pattern of having details like `price` as part of the deal. I think it makes it easier to deal with the swaps without having to make lookups of orders, by putting the key information on the swap deal itself.